reporting/accuracy/plot_kolmogorov_smirnov_pdf.cpp

```diff
@@ -0,0 +1,34 @@
+
+#include <iostream>
+#include <boost/math/tools/ulps_plot.hpp>
+#include <boost/core/demangle.hpp>
+#include <boost/math/distributions/kolmogorov_smirnov.hpp>
+
+using boost::math::tools::ulps_plot;
+
+int main() {
+    using PreciseReal = long double;
+    using CoarseReal = float;
+
+    boost::math::kolmogorov_smirnov_distribution<CoarseReal> dist_coarse(10);
+    auto pdf_coarse = [&, dist_coarse](CoarseReal x) {
+        return boost::math::pdf(dist_coarse, x);
+    };
+    boost::math::kolmogorov_smirnov_distribution<PreciseReal> dist_precise(10);
+    auto pdf_precise = [&, dist_precise](PreciseReal x) {
+        return boost::math::pdf(dist_precise, x);
+    };
```

Comment:
Haven't read the implementation, but if both of these are using a first-order approximation, is this a very meaningful ULPs plot? The vision for an ULPs plot is that the precise function is for all intents and purposes "correct".

Reply:
The "first order" is still an infinite sum with a convergence criterion, so I think the ULP plot will be useful for identifying aberrations.